spread/export_openstack_test.go

```diff
@@ -0,0 +1,89 @@
+package spread
+
+import (
+	"context"
+	"time"
+
+	gooseClient "github.com/go-goose/goose/v5/client"
+	"github.com/go-goose/goose/v5/glance"
+)
+
+var (
+	OpenstackName = openstackName
+)
+
+type (
+	OpenstackProvider   = openstackProvider
+	GlanceImageClient   = glanceImageClient
+	OpenstackServerData = openstackServerData
+)
+
+func MockOpenstackImageClient(opst *OpenstackProvider, newIC glanceImageClient) (restore func()) {
```

Comment:
newIC, newIC, but then "timeout" and "retry" shortly below.  Let's stick to the latter convention please and have "imageClient" and "computeClient" instead as these are more readable.

Reply:
Updated, Thank you!